runtime/src/lib.rs

```diff
@@ -5,7 +5,6 @@
 // Make the WASM binary available.
 #[cfg(feature = "std")]
 include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
-mod constants;
 mod types;
 
 use frame_system::limits::{
```

Comment:
suggest move down under `pub use frame_support` like done here https://github.com/substrate-developer-hub/substrate-parachain-template/blob/master/runtime/src/lib.rs#L33

Reply:
added to https://github.com/DataHighway-DHX/node/issues/231